app/code/Magento/StoreGraphQl/etc/graphql/di.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\StoreGraphQl\Model\Resolver\Store\ExtendedStoreConfigDataProvider">
+        <arguments>
+            <argument name="extendedConfigs" xsi:type="array">
+                <!-- Begin Store Configuration Web Default Pages -->
+                <item name="front" xsi:type="string">web/default/front</item>
```

Comment:
Yes, good point. 
Would be nice to avoid having coupling between di.xml and schema.graphqls and have just one declaration in schema.graphqls, but unfortunately config_path_with_underscore cannot be reliably converted to config/path/with_underscore.